docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,30 @@ weight: 35
 
 ## Unreleased
 
+## [v0.16.0](https://github.com/aws/eks-anywhere/releases/tag/v0.16.0)
+
+### Added
+- Workload clusters full lifecycle API support for CloudStack provider ([#2754](https://github.com/aws/eks-anywhere/issues/2754))
+- Enable proxy configuration for Bare Metal provider. ([#5925](https://github.com/aws/eks-anywhere/issues/5925))
+- Kubernetes 1.27 support ([#5929](https://github.com/aws/eks-anywhere/pull/5929))
+- Support for upgrades for clusters with pod disruption budgets ([5697](https://github.com/aws/eks-anywhere/pull/5697))
```

Comment:
the other items in the list have a `#` before the issue number. This one does not. might want to add it for consistency.